src/modules/site-v2/base/views/isotype.py

```diff
@@ -50,29 +51,27 @@ def isotype_page(isotype_name, release=None):
   try:
     disable_parent_breadcrumb = True
     isotype_strains = Strain.sort_by_strain( query_strains(isotype_name=isotype_name) )
+
     species = isotype_strains[0].species_name
     files = get_blob_list(MODULE_SITE_BUCKET_PHOTOS_NAME, species)
 
     image_urls = {}
     for s in isotype_strains:
       # Get images and thumbs for each strain
       for file in files:
-        start_idx = file.name.index('/')
-        end_idx = file.name.index('.')
-        file_name = file.name[start_idx+1:end_idx]
-        if s.strain != file_name:
+        if s.strain not in file.name:
           continue
         else:
```

Comment:
side note -- when we use early-return-guards, we can skip the `else:` and unindent the following code.